lms/models/lti_user.py

```diff
@@ -1,11 +1,20 @@
 from dataclasses import dataclass
-from typing import List
+from typing import List, Optional
 
 from lms.models.application_instance import ApplicationInstance
 from lms.models.h_user import HUser
 from lms.models.lti_role import LTIRole, RoleScope, RoleType
 
 
+@dataclass
+class LTI:
+    course_id: str
```

Comment:
As this are already namespace to `lti` using the regular naming, course and assignment.